frontend/__tests__/containers/ConfigStep/Board.test.tsx

```diff
@@ -84,13 +98,22 @@ describe('Board', () => {
   });
 
   it('should show board type when select board field value ', async () => {
-    setup();
+    const { getByRole, getByText } = setup();
```

Comment:
Avoid destructuring queries from `render` result, use `screen.getByRole` instead

Reply:
'getByText' is assigned a value but never used